docs/DeveloperGuide.md

```diff
@@ -288,30 +288,25 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `BizBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC1 - Add a person**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to add a new person by entering a command.
+2.  System displays a success message along with the details of the newly added person.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. There is an error in the user's command.
+    * 1a1 System shows an error message.
 
-  Use case ends.
+      Use case resumes at step 1.
```

Comment:
![image](https://github.com/user-attachments/assets/43a943c9-3c97-40e9-b5f8-939588bc0a84)
Could you remove the indentation here to align with Kenneth's format for "Use case resumes at step __"? Same for the other List All Contacts Use Case PR